server/src/main/java/com/objectcomputing/checkins/services/private_notes/PrivateNoteController.java

```diff
@@ -51,6 +53,7 @@ public PrivateNoteController(PrivateNoteServices privateNoteServices,
      * @return
      */
     @Post("/")
+    @RequiredPermission(Permissions.CAN_CREATE_PRIVATE_NOTE)
```

Comment:
If a user has the `ELEVATED_ACCESS` permission, they are prohibited from accessing this endpoint unless that user also has the `CAN_CREATE_PRIVATE_NOTE` permission. This goes for all the other endpoints annotated with `@RequiredPermission` as well.

Reply:
As per the conversation at stand up, we are going to keep all permissions separate and address permission dependencies at a later time.